tweepy/error.py

```diff
@@ -30,4 +31,23 @@ class RateLimitError(TweepError):
     """Exception for Tweepy hitting the rate limit."""
     # RateLimitError has the exact same properties and inner workings
     # as TweepError for backwards compatibility reasons.
-    pass
+
+    def reset_on(self):
+        '''Returns the remaining time in seconds for the rate limit to be replenished.
+
+        Uses the 'x-rate-limit-reset' header to calculate the countdown in seconds,
+        for the rate limit to be replenished. When the 'x-rate-limit-reset' header
+        is not present it returns the default reset time.
```

Comment:
nit: Might be useful to include what the actual default is in the docstring, i.e. 15 minutes

Reply:
adressed in c1c0139